### ISSUE TITLE
[DetailList] Screen reader doesn't announce searched results at filter edit fields.

### ISSUE THREAD

**Issue by anmohanty123** [microsoft/fluentui#23042]
Can we get this issue fixed please? Its blocking our accessibility testing

This is an issue with our example implementation rather than DetailsList itself, since the filter input isn't built into the component.

To do this right, we'd need to set up a timeout to debounce updates while typing, and only fire a live region with the number of updated results after about a half second of inactivity. I'll defer to @miroslavstastny on whether we should implement that in this example page, since it's not directly a part of the DetailsList.

_Originally posted by @smhigley in https://github.com/microsoft/fluentui/issues/18293#issuecomment-869887795_

**Comment by gouttierre** [comment]
@anmohanty123 - Thanks for filing this issue with us. To set expectations the developers will review this issue once capacity allows.

@smhigley - would you have any updates on this particular [issue](https://github.com/microsoft/fluentui/issues/18293#issuecomment-869887795)? 

cc @khmakoto
